### PR TITLE
[PH] Update initial API Node Op Mode documentation.

### PR DESCRIPTION
**Pull request by oschwaldp-oci**

### CODE REVIEW

**Comment by stephenpdeos** [comment]
@wanderingbort and @bhazzard I think you two may be the best reviewers for this. I know Brian collected a lot of details around node operation modes and practices in roundtables historically.

**Review by bhazzard — APPROVED** [review]

**Review by greg7mdp — APPROVED** [review]
Exciting, I can't wait to use this!

File: tests/performance_tests/ph_op_modes.md [hunk 1]
```diff
@@ -34,17 +34,29 @@ The perfomance_test.py and performance_test_basic.py are currently configured to
...
 
+Determine the maximum throughput (transactions per second) of a specified transaction type (token transfer, read only, new account, etc.) the single producer system can sustain.
+The validation node is used by the performance harness framework to monitor and query information about the performance and progress of the producer node as it processes API node transactions,
+so as to not adversely affect the producer's ability to process transactions by answering queries from the performance harness concurrently.
+
 ### Configuration
 
+Additional node included over the base configuration. This additional node will have the http_plugin enabled and will be the recipient of all generated transaction traffic.
+Configure unlimited `max_bytes_in_flight` and `max_messages_in_flight` and `max_requests_in_flight`.
+
```
[the remark this answers is not in the document]
Comment (oschwaldp-oci) [reply tests/performance_tests/ph_op_modes.md:56]: Currently the Performance Harness already defaults the `http-max-response-time-ms` to -1 (in `performance_test_basic.py` argument parser), but if you'd like it to be directly called out here, I can add it.

[the remark this answers is not in the document]
Comment (greg7mdp) [reply tests/performance_tests/ph_op_modes.md:56]: No, don't add it if it is not useful, I was just wondering.

[the remark this answers is not in the document]
Comment (oschwaldp-oci) [reply tests/performance_tests/ph_op_modes.md:60]: Addressed: [575d759](https://github.com/AntelopeIO/leap/pull/1056/commits/575d7596e1dc0918e280416dd744c3a21e645ad2)

**Review by greg7mdp — APPROVED** [review]

**Pull request merged**